database/GenerateDataPostgres.js

```diff
@@ -0,0 +1,37 @@
+const fs = require('fs');
+const faker = require('faker');
+
+const wStream = fs.createWriteStream('./postgresUsers.csv');
+
+let userId = 1;
+const users = [];
+const start = new Date().getTime();
+const n = 20000000;
+
+for (let i = 0; i <= 500; i += 1) {
+  users.push(faker.name.findName());
+}
+for (let j = 0; j <= n; j += 1) {
+  const header = 'id, username';
+  const values = `${userId},${users[Math.floor(Math.random() * users.length)]}`;
+  userId += 1;
+  if (j === 0) {
+    wStream.write(`${header}\n`, (err) => {
+      if (err) console.log(err);
+      if (j % 20000000) {
```

Comment:
(j % 20000000) will be truthy whenever j is not 20000000, 0, or a multiple of it--is this what you want?